xxhash.c

```diff
@@ -297,12 +297,15 @@ XXH_PUBLIC_API unsigned XXH_versionNumber (void) { return XXH_VERSION_NUMBER; }
 /* *******************************************************************
 *  32-bit hash functions
 *********************************************************************/
+#ifdef __cplusplus
+#  define const constexpr
```

Comment:
This seems to break the g++ compilation test.

What was the expected benefit from this change ?

Reply:
Gah, I was testing something and forgot to change it back.